gradle.properties

```diff
@@ -105,8 +105,8 @@ jetbrains.compose.compiler.version=1.5.14.1
 #   artifactRedirection.version.<redirectionGroup>.<redirectionModule>
 #   covers all sub-module projects
 #   (artifactRedirection.version.androidx.compose covers androidx.compose.*:*)
-artifactRedirection.version.androidx.compose=1.8.0-rc03
-artifactRedirection.version.androidx.compose.material.material-navigation=1.8.0-rc03
+artifactRedirection.version.androidx.compose=1.8.0
+artifactRedirection.version.androidx.compose.material.material-navigation=1.8.0
```

Comment:
We redirect to stable version, but we shouldn't release CMP version as stable for this library